back/config/schemas/settings.schema.json.erb

```diff
@@ -1179,6 +1179,18 @@
         }
       },
 
+      "posthog_user_tracking": {
```

Comment:
Did not add the tracking of back-end events as it is for admins. Reasoning:
- Felt too complicated as a proof of concept
- Needed to be based on cookie consent which I am not sure how to do with back-end tracking.